src/redux/reducers/todosReducer.js

```diff
@@ -1,13 +1,32 @@
+import { ADD_TODO, IS_ADDING } from "redux/actions/actionTypes"
+import { formDate } from "utils/formDate"
 export const initialState = {
-  todosItems: [
-    { text: "Eat", completed: true },
-    { text: "Sleep", completed: false },
-  ],
+  todoItems: [],
   visibilityFiler: "SHOW_ALL",
+  isAdding: false,
 }
 
 export default function todosReducer(state = initialState, action) {
   switch (action.type) {
+    case ADD_TODO:
+      const { text } = action.payload
+      return {
+        ...state,
+        todoItems: [
+          ...state.todoItems,
+          {
+            text: text,
+            createdAt: formDate(new Date()),
+            completed: false,
```

Comment:
We may have the value from our Payload means you may need to add completed inside your action payload as well.